recipes/python_rc_dummy/meta.yaml

```diff
@@ -0,0 +1,25 @@
+{% set name = "python_rc_dummy" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+test:
+  commands:
+  - echo "Keep conda-smithy happy."
+
+about:
+  home: https://github.com/conda-forge/python_rc_dummy-feedstock
+  summary: 'Dummy package to force selection of python_rc channel when channel_priority is set to strict'
+  description: 'See https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/280'
+  license: Unlicense
+
+extra:
+  recipe-maintainers:
+    - '0xBE7A'
+    - xhochy
```

Comment:
Happy to be a maintainer; alternatively we could also set `conda-forge/python` as maintainer here. I'll let @isuruf decide.